app/models/host_status/configuration_status.rb

```diff
@@ -78,7 +78,7 @@ def to_status(options = {})
     end
 
     def relevant?
-      host.configuration?
+      host.configuration? || last_report.present?
```

Comment:
Maybe we could expose a setting? as a way saying I don't care about proxies? I have no idea how common this usage case is.